### ISSUE TITLE
Add monospace font for Linux

### ISSUE THREAD

**Issue by panda-z** [olOwOlo/hugo-theme-even#106]
The current code font family does not contains Linux font.

Can we add monospace font for Linux? Candidates are `DejaVu Sans Mono` and `Bitstream Vera Sans Mono`.

See for example: http://www.logz.org/art-416-tit-Best-font-family-for-linux-mac-and-linux

**Comment by olOwOlo** [comment]
Sorry for the delay, I don't know much about the font family. Do you mean to modify this?

https://github.com/olOwOlo/hugo-theme-even/blob/b11fe83ad27da925f76d75386557ab2abcd0bd9d/src/css/_variables.scss#L58

**Comment by panda-z** [comment]
That's OK. I just found that I messed up my fontconfig file in Linux so that the font for code snippet looks weird. After modifying the fontconfig configuration file, the font is normal now.

What I mean is font family for code snippets in this line:
https://github.com/olOwOlo/hugo-theme-even/blob/b11fe83ad27da925f76d75386557ab2abcd0bd9d/src/css/_variables.scss#L196

No need for two `Consolas` right?

And also, `Consolas` for Windows, `Menlo` for Mac, maybe add `"Bitstream Vera Sans Mono"` and/or `"DejaVu Sans Mono"` for Linux.
